File: js/get-localized-content/IgnoreFallback.js
// DocSection: language_fallbacks_ignore
// Tip: Find more about JS/TS SDKs at https://kontent.ai/learn/javascript
const KontentDelivery = require('@kontent-ai/delivery-sdk');

const deliveryClient = KontentDelivery.createDeliveryClient({
  environmentId: '975bf280-fd91-488c-994c-2f04416e5ee3',
});

const response = await deliveryClient.items()
  .languageParameter('es-ES')
  .equalsFilter('system.language', 'es-ES')
  .toPromise();
// EndDocSection

File: ruby/get-localized-content/IgnoreFallback.rb
# DocSection: language_fallbacks_ignore
# Tip: Find more about Ruby SDKs at https://kontent.ai/learn/ruby
require 'delivery-sdk-ruby'

delivery_client = Kentico::Kontent::Delivery::DeliveryClient.new project_id: '975bf280-fd91-488c-994c-2f04416e5ee3'
delivery_client.items(['system.language'.eq('es-ES')])
               .language('es-ES')
               .execute do |response|
                 items = response.items
               end
# EndDocSection
